src/migrations/20201021001745-create-payoutMethod.js

```diff
@@ -0,0 +1,37 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('payoutMethod', {
```

Comment:
What is this entity for?

Reply:
I've updated the comment to explain a bit more, but this is intended to emulate a polymorphic entity for different payment options.

Reply:
@abelosorio This is an implementation of a Polymorphic Association: https://sequelize.org/master/manual/polymorphic-associations.html

This model is a relationship between Business and a generic Payout Method. You don't need to know what kind of payout method it is when using it (Mercado Pago, Bank Account, etc). Each Payout Method can have its own data in its own table, and this table is how we know which kind of payout method we're using.

**It's all explained in the main text of this PR :smile: **

Reply:
You guys are evil.